nixos-modules/hydra.nix

```diff
@@ -1,3 +1,4 @@
+{ self, ... }:
```

Comment:
Please don't pass in `self`, because then this file becomes non-extensible to new platforms

Reply:
I dropped this commit. Because than I don't have access to pkgs anymore and need a second nixos module with the _file hack again, which kind of defeats the point the change.